src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -321,6 +323,68 @@ def fuzz(self,
                                 artifact_prefix, additional_args, extra_env)
 
 
+class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
+  """libFuzzer runner (when Fuchsia is the target platform)."""
+
+  SSH_RETRIES = 10
+  SSH_WAIT = 2
```

Comment:
nit: can you add the suffix '_SECONDS' to this attribute please?
Can come in another CL.